command/service_cmd.go

```diff
@@ -0,0 +1,217 @@
+
+	resp, err := cli.Update(nil, &req)
+	if err != nil {
+		fmt.Printf("%s\n", err.Error())
+		os.Exit(1)
+	}
+
+	printVerboseMessage("Updating Service. RequestID: " + resp.RequestId)
+	fmt.Println("RequestID: " + resp.RequestId)
+}
+
+// DeleteServiceAction updates a service in OpsGenie
+func DeleteServiceAction(c *gcli.Context) {
+	cli, err := NewServiceClient(c)
+	if err != nil {
+		os.Exit(1)
+	}
+
+	req := service.DeleteRequest{}
+	if val, success := getVal("id", c); success {
+		req.Id = val
+	}
+
+	printVerboseMessage("Delete Service Request Created. Sending to Opsgenie...")
+
+	resp, err := cli.Delete(nil, &req)
+	if err != nil {
+		fmt.Printf("%s\n", err.Error())
+		os.Exit(1)
+	}
+
+	printVerboseMessage("Deleting Service. RequestID: " + resp.RequestId)
+	fmt.Println("RequestID: " + resp.RequestId)
+}
+
+// GetServiceAction updates a service in OpsGenie
+func GetServiceAction(c *gcli.Context) {
+	cli, err := NewServiceClient(c)
+	if err != nil {
+		os.Exit(1)
+	}
+
+	req := service.GetRequest{}
+	if val, success := getVal("id", c); success {
+		req.Id = val
+	}
+
+	printVerboseMessage("Get Service Request Created. Sending to Opsgenie...")
+
+	resp, err := cli.Get(nil, &req)
+	if err != nil {
+		fmt.Printf("%s\n", err.Error())
+		os.Exit(1)
+	}
+
+	printVerboseMessage("Getting Service. RequestID: " + resp.RequestId + resp.Service.Id)
+	fmt.Println(resp)
+	fmt.Println("RequestID: " + resp.RequestId + "***" + resp.Service.Id)
+	outputFormat := strings.ToLower(c.String("output-format"))
+	switch outputFormat {
+	case "yaml":
+		output, err := resultToYAML(resp.Service)
+		if err != nil {
+			fmt.Printf("%s\n", err.Error())
+			os.Exit(1)
+		}
+		fmt.Printf("%s\n", output)
+	default:
+		isPretty := c.IsSet("pretty")
+		output, err := resultToJSON(resp.Service, isPretty)
+		if err != nil {
+			fmt.Printf("%s\n", err.Error())
+			os.Exit(1)
+		}
+		fmt.Printf("%s\n", output)
+	}
+}
+
+// ListServiceAction updates a service in OpsGenie
+func ListServiceAction(c *gcli.Context) {
+	cli, err := NewServiceClient(c)
+	if err != nil {
+		os.Exit(1)
+	}
+
+	req := service.ListRequest{}
+	if val, success := getVal("limit", c); success {
+		limit, err := strconv.ParseUint(val, 10, 64)
+		if err != nil {
+			os.Exit(2)
+		}
+		req.Limit = int(limit)
+	}
+
+	if val, success := getVal("offset", c); success {
+		offset, err := strconv.Atoi(val)
+		if err != nil {
+			os.Exit(2)
+		}
+		req.Offset = offset
+	}
+
+	printVerboseMessage("Get Service List Request Created. Sending to Opsgenie...")
+
+	resp, err := cli.List(nil, &req)
+	if err != nil {
+		fmt.Printf("%s\n", err.Error())
+		os.Exit(1)
+	}
+
+	printVerboseMessage("Getting Services. RequestID: " + resp.RequestId)
+	fmt.Println("RequestID: " + resp.RequestId)
+	outputFormat := strings.ToLower(c.String("output-format"))
+	printVerboseMessage("Got Services successfully, and will print as " + outputFormat)
+	switch outputFormat {
```

Comment:
you can use `renderResponse` method which does this check and prints the response